### PR TITLE
feat: adjust the display of publications

### PR DESCRIPTION
**Pull request by mpro7**
resolves DSP-3764

### CODE REVIEW

**Review by BalduinLandolt — APPROVED** [review]
didn't run it myself, but code makes sense and looks good

File: services/metadata/backend/data/fagottino.json [hunk 1]
```diff
@@ -71,10 +71,18 @@
...
-            "Agrell, Donna and Domínguez, Áurea, “Fagottini and tenoroons: small, forgotten giants”, The Double Reed 42/2 (2019), 57–68.",
-            "Agrell, Donna and Domínguez, Áurea, “Small-sized bassoons from the eighteenth and nineteenth centuries under investigation”, Glareana 67/2 (2018), 58–74. Full text available: http://www.gefam.ch/uploads/Glareana_67_2018_2.pdf.",
-            "Domínguez, Áurea,“Fagottino in the Philharmonie de Paris: Researching the largest collection of small-sized bassoons”, in: Glareana 68/2, (2019), 6–19.",
-            "Domínguez, Áurea. “The Romantic bassoon understood by 19th century performers.” In Geschichte, Bauweise und Repertoire des Fagotts. Michaelstein: Wissner-Verlag (2020), 153–171."
+            {
+                "text": "Agrell, Donna and Domínguez, Áurea, “Fagottini and tenoroons: small, forgotten giants”, The Double Reed 42/2 (2019), 57–68."
+            },
+            {
+                "text": "Agrell, Donna and Domínguez, Áurea, “Small-sized bassoons from the eighteenth and nineteenth centuries under investigation”, Glareana 67/2 (2018), 58–74. Full text available: http://www.gefam.ch/uploads/Glareana_67_2018_2.pdf."
```
[the remark this answers is not in the document]
Comment (mpro7) [reply services/metadata/backend/data/fagottino.json:78]: This is also dead-link.

File: services/metadata/backend/data/mark16.json [hunk 2]
```diff
@@ -66,32 +66,188 @@
...
+                "text": "Clivaz, Claire, “Editer le Nouveau Testament à l’heure du numérique”, Bulletin de l’Académie des Sciences humaines et sociales 3 (2021), p. 37-40."
+            },
+            {
+                "text": "Monier, Mina, “Mark’s Ending in the Digital Age: Paratextual Evidence, New Findings and Transcription Challenges”, Postscripts: The Journal of Sacred Texts, Cultural Histories, and Contemporary Contexts 12 (2021/1), p. 75-98."
+            },
+            {
+                "text": "Clivaz, Claire, and Garrick V. Allen (eds.), “Ancient Manuscripts and Virtual Research Environments” Special issue, Classics@ 18 (2021)",
+                "url": [
+                    {
+                        "__type": "URL",
```
[the remark this answers is not in the document]
Comment (mpro7) [reply services/metadata/backend/data/mark16.json:125]: Due to they way how we have written our CSS here, I had to truncated it manually. If there will be more cases like this, I will need to redesign the project page ...

File: services/metadata/backend/data/woposs.json [hunk 3]
```diff
@@ -70,27 +70,141 @@
...
+            {
+                "text": "Dell'Oro, Francesca (2023, accepted). «Corpus parallèles et apprentissage des langues anciennes: les Évangiles comme corpus multilingue pour apprendre le grec ancien et le latin (avec un focus sur la modalité)» in La traduction et son processus didactique, edited by Daria Zalesskaya, Cahiers du CLSL - Lausanne."
+            },
+            {
+                "text": "Dell’Oro, Francesca; Bermúdez Sabel, Helena; Marongiu, Paola (2022). “Pygmalion in the classroom: a tool to draw lexicographic diachronic maps and their application to didactics” in La lexicografía didáctica: Reflexiones y recursos orientados al aprendizaje de lenguas, edited by Manuel Márquez Cruz, and Viviane Ferreira Martins. Madrid: Guillermo Escolar Editor, 35–48. [Preprint:",
+                "url": [
+                    {
+                        "__type": "URL",
+                        "type": "URL",
+                        "url": "https://woposs.unine.ch/outputs/DellOro_BermudezSabel_Marongiu_Pygmalion_Lexicografia.pdf]",
```
[the remark this answers is not in the document]
Comment (mpro7) [reply services/metadata/backend/data/woposs.json:135]: The other way around, `text` with `]` not `url` ;)

File: services/metadata/backend/data/big-data-in-agriculture.json [hunk 4]
```diff
@@ -52,8 +52,28 @@
         ],
         "name": "Big Data in Agriculture: The Making of Smart Farms",
```
[the remark this answers is not in the document]
Comment (mpro7) [reply services/metadata/backend/data/big-data-in-agriculture.json:53]: As agreed, we keep it as it is.

File: services/metadata/backend/data/participation-social.json [hunk 5]
```diff
@@ -52,8 +52,28 @@
             }
         ],
         "publications": [
```
[the remark this answers is not in the document]
Comment (mpro7) [reply services/metadata/backend/data/participation-social.json:54]: As agreed, we keep it as it is.

File: services/metadata/backend/data/lenzburg.json [hunk 6]
```diff
@@ -70,8 +70,20 @@
         ],
         "name": "Tonbandsammlung Folkfestival auf der Lenzburg",
```
[the remark this answers is not in the document]
Comment (mpro7) [reply services/metadata/backend/data/lenzburg.json:71]: As agreed, we keep it as it is.

**Pull request merged**